### DISCUSSION TITLE
Open Calendar Macro

### DISCUSSION THREAD

**Discussion by thecodex101** [vigoren/foundryvtt-simple-calendar#194] [Q&A] [answered]
Hi, 

I use the Trigger Happy Modual and want to set a button that will open the Calnedar in Full Mode for the Players to look at. Is there a Macro way to open the Calendar specifically in Full Mode for Players? 

Cheers

Ian

**Comment by vigoren** [comment, marked as answer]
I apologize for answering this so late, I missed the notification.

There is indeed a macro way to open the calendar!

`SimpleCalendar.api.showCalendar();` that will open it up in full mode.

**Reply by cryptiklemur** [reply]
Is there not a way to close it? I didn't see a method for it in the docs
